Jenkinsfile

```diff
@@ -2,7 +2,13 @@
 
 buildMvn {
   publishModDescriptor = 'yes'
-  doApiDoc = 'yes'
+
+  doApiDoc = true
+  doApiLint = true
+  apiTypes = 'RAML'
+  apiDirectories = 'ramls'
+  apiExcludes = 'instance-storage.raml holdings-storage.raml'
```

Comment:
Why are these RAML files excluded from the documentation?

Reply:
Cause they have errors during the linting. Im not an owner of these APIs and not sure that can fix it without breaking changes. I believe it's better to have an additional task for it.

Reply:
> Cause they have errors during the linting. Im not an owner of these APIs and not sure that can fix it without breaking changes. I believe it's better to have an additional task for it.

That seems like a strange decision to me. By excluding them, it effectively hides the errors, meaning they are less likely to be resolved.

What are the errors that you consider would require breaking changes to resolve?